src/module.c

```diff
@@ -4803,6 +4807,26 @@ int moduleAllDatatypesHandleErrors() {
     return 1;
 }
 
+/* Returns 0 if there's at least one registered data type that did not declare
+ * REDISMODULE_OPTIONS_HANDLE_REPL_ASYNC, in which case diskless async loading should be avoided
+ * because module doesn't know there can be traffic during database full resynchronization. */
+int moduleAllDatatypesHandleReplAsync() {
```

Comment:
```suggestion
 * REDISMODULE_OPTIONS_HANDLE_REPL_ASYNC_LOAD, in which case diskless async loading should be avoided
 * because module doesn't know there can be traffic during database full resynchronization. */
int moduleAllDatatypesHandleReplAsyncLoad() {
```